OrbitGl/PresetsDataView.cpp

```diff
@@ -153,13 +199,45 @@ void PresetsDataView::DoFilter() {
 
 void PresetsDataView::OnDataChanged() {
   indices_.resize(presets_.size());
+  modules_.resize(presets_.size());
   for (size_t i = 0; i < presets_.size(); ++i) {
     indices_[i] = i;
+    std::vector<ModuleView> modules;
+    for (const auto& pair : presets_[i]->preset_info().path_to_module()) {
+      modules.push_back(
+          ModuleView(Path::GetFileName(pair.first), pair.second.function_hashes_size()));
+    }
+    modules_[i] = modules;
   }
 
   DataView::OnDataChanged();
 }
 
+bool PresetsDataView::GetDisplayColor(int row, int /*column*/, unsigned char& red,
```

Comment:
What about returning a `std::optional<Color>`? If I'm not mistaken we have a Color type.

Reply:
Ahhh. I just saw, this is an override. Never mind...

Reply:
I agree, we should return Color, added a TODO for now